crosswords/Views/CrosswordCellView.swift

```diff
@@ -44,7 +44,9 @@ struct CrosswordCellView: View, Equatable {
                         .frame(alignment: .center)
                 }
             }
-            if symbol >= 1000 {
+            if symbol >= 10000 {
+                Color.gray
```

Comment:
please test in dark mode and light mode. Also, what if the cell is supposed to be highlighted?

Reply:
for example, if the cell is focused, we don't want it to be a gray color. If error tracking is enabled, it also should appear a red shade if the cell's value is incorrect.